lib/service/skills/IOccupationsData.dart

```diff
@@ -0,0 +1,11 @@
+// TODO: Fix the return values of all the
+
+import 'package:illinois/model/occupation/Occupation.dart';
+
+abstract class IOccupationsData /* with Service */ {
```

Comment:
I'm not sure if this class is needed since only `OccupationsService` will be performing these operations.